styles.css

```
@@ -1,6 +1,40 @@
 @charset 'UTF-8';
 
-h1 {
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 2.4rem;
```

```suggestion
  padding: 1rem 2.4rem;
```
O que acha, sobre adicionar um padding horizontal para ficar mais parecido com o nosso template?